firmware/library/L1_Drivers/pin_configure_test.cpp

```diff
@@ -0,0 +1,133 @@
+#include <assert.h>
+#include <cstdio>
+#include "L0_LowLevel/LPC40xx.h"
+#include "L5_Testing/testing_frameworks.hpp"
+#include "L1_Drivers/pin_configure.hpp"
+
```

Comment:
Add comment about how you are going about testing this via side effect, based on your design document.